### PR TITLE
Doc415 addfavicon

### PR DESCRIPTION
**Pull request by Doc415**
Added C# Academy favicon

### CODE REVIEW

**Comment by Doc415** [comment]
Updated "Submit PR form" in community projects page.
Since i cant test it locally, this is a blind shot.

File: TCSA.V2/Components/App.razor [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" href="bootstrap/bootstrap.min.css" />
     <link rel="stylesheet" href="app.css" />
     <link rel="stylesheet" href="TCSA.V2.styles.css" />
-    <link rel="icon" type="image/png" href="favicon.png" />
+    <link rel="icon" type="image/x-icon" href="/favicon.ico">
```
Comment (TheCSharpAcademy) [inline TCSA.V2/Components/App.razor:11]: Hello! At the moment I see this icon. I was expecting the C# Academy icon?

File: TCSA.V2/Components/Dashboard/Pages/Community.razor [hunk 2]
```diff
@@ -253,6 +265,7 @@
 
     private async Task SubmitIssue()
     {
+        await JsRuntime.InvokeVoidAsync("scrollToTop");
```
Comment (TheCSharpAcademy) [inline TCSA.V2/Components/Dashboard/Pages/Community.razor:268]: No JS please. 😁

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

File: TCSA.V2/Components/App.razor (hunk 1 above)
Comment (Doc415) [reply TCSA.V2/Components/App.razor:11]: I took it from the old site. Thats what i see in old site as favicon.

Comment (Doc415) [reply TCSA.V2/Components/App.razor:11]: image
 this is from old site.

So I dont have any other file to put. If you can send me the icon file i ll replace it.

File: TCSA.V2/Components/Dashboard/Pages/Community.razor (hunk 2 above)
Comment (Doc415) [reply TCSA.V2/Components/Dashboard/Pages/Community.razor:268]: sure :)

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
@Doc415 This PR still contains too many changes. Could you please modify it so it only contains the changes relevant to the favicon?

**Comment by TheCSharpAcademy** [comment]
@Doc415 Any updates?

**Comment by Doc415** [comment]
> @Doc415 Any updates? 😊

İ have created seperated pr  for each issue waitin for review :)

Pr no #189 and #190

**Comment by Doc415** [comment]
> @Doc415 Any updates? 😊

I have created seperated pr for each issue waiting for review :)

Pr #189 and #190

**Pull request closed without merging**